var/spack/repos/builtin/packages/sw4lite/package.py

```diff
@@ -52,6 +52,14 @@ def build_targets(self):
             cxxflags.append('-DSW4_CROUTINES')
             targets.append('ckernel=yes')
 
+        if '+cuda' in self.spec:
+            targets.append('NVCC = {0}'.format(
+                self.spec['cuda'].prefix.bin.nvcc))
+            targets.append('HOSTCOMP = {0}'.format(self.compiler.cxx))
+            targets.append('MPIPATH= {0} '.format(self.spec['mpi'].prefix))
+            targets.append('gpuarch= {0}'.format(self.cuda_flags(cuda_arch)))
+            targets.append('MPIINC = {0}'.format(self.spec['mpi'].include))
```

Comment:
Did you mean `prefix.include`? It might be best to use `headers.directories[0]` here.

Reply:
Ooh, did not know that was a thing. That will make a lot of other things a lot easier. Fixed and thanks.